### PR TITLE
functests: utils: add ExecAndLogCommandWithStderr

### PR DESCRIPTION
**Pull request by fromanirh**
Add a new helper to make it easier and less complex to get
the stderr of a command execution.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: functests/utils/utils.go [hunk 1]
```diff
@@ -43,3 +44,32 @@ func ExecAndLogCommand(name string, arg ...string) ([]byte, error) {
 	}
 	return out, err
 }
+
+func ExecAndLogCommandWithStderr(name string, arg ...string) ([]byte, []byte, error) {
```
Comment (cynepco3hahue) [inline functests/utils/utils.go:48]: Can you update `ExecAndLogCommand` to use your new command?

Comment (fromanirh) [reply functests/utils/utils.go:48]: sure, on it

**Comment by cynepco3hahue** [comment]
Infra problems
/retest

**Pull request merged**